v21.1/topology-duplicate-indexes.md

```diff
@@ -1,683 +0,0 @@
----
```

Comment:
I don't understand the deletion, is it because we are 301 permanently redirecting away from this?

Reply:
That is correct.